hack/e2e.sh

```diff
@@ -7,12 +7,12 @@ fi
 if [ -z "$SUITE" ] || [ "$SUITE" == "enduser" ]; then
   echo "Running end user e2e tests"
   # Login as osadmin to simulate a regular user
-  password=$(awk '/^  adminPasswd:/ { print $2 }' <_data/containerservice.yaml)
+  password=$(awk '/^  NormalUserPasswd:/ { print $2 }' <_data/containerservice.yaml)
   fqdn=$(awk '/^  fqdn:/ { print $2 }' <_data/containerservice.yaml)
-  export KUBECONFIG=_data/_out/osadmin.kubeconfig
-  # oc login is going to create the osadmin.kubeconfig for us
-  oc login $fqdn --username osadmin --password $password --insecure-skip-tls-verify=true
-  go test ./test/e2e -test.v -ginkgo.v -ginkgo.focus="\[EndUser\]" -ginkgo.noColor -ginkgo.randomizeAllSpecs -tags e2e -kubeconfig ../../_data/_out/osadmin.kubeconfig 2>&1 | tee end-user.log
+  export KUBECONFIG=_data/_out/normal.kubeconfig
```

Comment:
s/normal/enduser/

Reply:
Do we want enduser rename across the PR for this user?  I chose normal but I don't have any preference.